tests/integration/run.bats

```diff
@@ -219,3 +219,21 @@ EOF
 	[ ${#lines[@]} -eq 1 ]
 	[[ ${lines[0]} = "exec /run.sh: no such file or directory" ]]
 }
+
+# https://github.com/opencontainers/runc/issues/4688
+@test "runc run check default home" {
+	requires root
```

Comment:
Does it require root?

Reply:
IMHO it does not; instead, it should use
```bash
        [ $EUID -ne 0 ] && requires rootless_idmap
```

Reply:
Thanks!